README.rst

```diff
@@ -62,3 +80,15 @@ optimally configured.
 .. code-block:: console
 
     $ xbrl_extract {path_to_filings} {path_to_database} --workers {number_of_processes} --batch-size {filings_per_batch}
+
+There are also several options included for extracting metadata from the taxonomy.
+First is the ``--save-datapackage`` command to save a
+`frictionless datapackage <https://specs.frictionlessdata.io/data-package/>`__ as a
+json file. There is also the ``--metadata-path``, which writes taxonomy metadata to
+a json file that groups metadata by table name. See the ``arelle_interface`` module
+for more info on the extracted metadata. To create both of these files using the
+example filings and taxonomy, run the following command.
+
+.. code-block:: console
+
+    $ xbrl_extract examples/ferc1-2021-sample.zip {path_to_database} --taxonomy examples/taxonomy.zip --archive-path taxonomy/form1/2021-01-01/form/form1/form-1_2021-01-01.xsd --metadata-path metadata.json --save-datapackage datapackage.json
```

Comment:
Let's break this into a multi-line command for readability, and use the same file naming conventions that we have in the PUDL repo, so that they feel familiar if someone runs across them both:

```
$ xbrl_extract examples/ferc1-2021-sample.zip ferc1-2021-sample.sqlite \
    --taxonomy examples/taxonomy.zip \
    --archive-path taxonomy/form1/2021-01-01/form/form1/form-1_2021-01-01.xsd \
    --metadata-path ferc1_xbrl_taxonomy_metadata.json \
    --save-datapackage ferc1_xbrl_datapackage.json
```

This command surprised me. I did not understand earlier on that you were talking about two (potentially?) different zipfiles for the filings themselves and the taxonomy. Do they have to be separate? Are they allowed to be in the same zipfile? How is it arranged in the PUDL datastore archives?

Reply:
Good call, I've updated all the long commands to be multi-line